bin/bashcov

```diff
@@ -13,8 +13,33 @@ coverage = runner.result
 
 require "simplecov"
 
-SimpleCov.command_name Bashcov.name
+SimpleCov.coverage_dir(ENV["COVERAGE_DIR"]) if ENV["COVERAGE_DIR"]
+
+if ENV["COVERAGE_NAME"]
+  SimpleCov.command_name ENV["COVERAGE_NAME"]
+else
+  SimpleCov.command_name Bashcov.name
+end
+
+reports = Bashcov.options.reports
+unless reports.empty?
+  formatters = [
+    SimpleCov::Formatter::HTMLFormatter
+  ]
+  reports.each do |r|
+    require r[:require]
+    formatters.push(r[:formatter].split("::").inject(Object) { |a, e| a.const_get e })
+  end
+  SimpleCov.formatters = formatters
+end
```

Comment:
I don't think we need to reimplement that in Bashcov. I just tried [this](https://github.com/albfan/git-ignore/commit/ff945c25ca997331377f761af2a6d9a7a0774302) and it builds fine on Travis. However it [doesn't seem to push coverage results](https://coveralls.io/github/infertux/git-ignore) even [with your original code](https://travis-ci.org/infertux/git-ignore/jobs/87130726). So I must have misconfigured Coveralls somehow. Could you try something along the lines of [this](https://github.com/albfan/git-ignore/commit/ff945c25ca997331377f761af2a6d9a7a0774302) in your own repo and see if that works?